src/checks/y_check_branch_coverage.clas.abap

```diff
@@ -18,7 +18,7 @@ CLASS y_check_branch_coverage IMPLEMENTATION.
     settings-threshold = 100.
     settings-is_threshold_reversed = abap_true.
     settings-disable_on_prodcode_selection = abap_true.
-    settings-disable_on_testcode_selection = abap_true.
+    settings-disable_on_testcode_selection = abap_false.
```

Comment:
you just enabled the testcode selection and not disabled anything.

you need: 
settings-apply_on_test_code = abap_false.

Reply:
oh, well observed. Thanks for it!